ui/components/MeshAdapterConfigComponent.js

```diff
@@ -138,6 +152,32 @@ class  MeshAdapterConfigComponent extends React.Component {
     );
   };
 
+  setAdapterStates = () => {
+    const { meshAdapters } = this.state;
+    const initialAdapterStates = {};
+
+    meshAdapters.forEach((adapter) => {
+      const lable = adapter.name.toUpperCase()
+      initialAdapterStates[lable] = STATUS.UNDEPLOYED;
+    });
+
+    this.setState({
+      adapterStates : initialAdapterStates,
+    });
+  }
+
+  getStatusColor = (status) => {
```

Comment:
Some pretty good stuff here, @Yana-Gupta. 🙂

Reply:
We'll want to further centralize this states and their colors after this PR merges.

WE're about to start on a big journey into Connections and management of objects that MeshSync discovers.

Reply:
Thanks! I am glad to hear about the plan to centralise the states and eager to contribute and help with the upcoming journey.